src/types/onyx/Request.ts

```diff
@@ -1,12 +1,19 @@
 import {OnyxUpdate} from 'react-native-onyx';
 
-type Request = {
+type OnyxData = {
```

Comment:
This type is also being created in this PR -> https://github.com/Expensify/App/pull/28031/files#diff-8749e183a5917b6f0e40dd771d46e94a1f4e0201fe8ca950c5814919d7409237

cc @blazejkustra

Reply:
^^ @blazejkustra